java-contentwarehouse/README.md

```diff
@@ -0,0 +1,201 @@
+# Google Document AI Warehouse Client for Java
+
+Java idiomatic client for [Document AI Warehouse][product-docs].
```

Comment:
Where is product doc?

Reply:
I think it is missing [here](https://github.com/googleapis/google-cloud-java/pull/8563/files#diff-74dbab4bda95728466e773ea1936a5ad82ce62ecac67f992a5554b1720ff8e2e) as well. The command I ran to generate this library is : 
`python3.9 new-client.py generate \
  --api_shortname=contentwarehouse \
  --proto-path=google/cloud/contentwarehouse \
  --name-pretty="Document AI Warehouse" \
  --product-docs="https://cloud.google.com/document-warehouse/docs/overview" \
  --api-description="Document AI Warehouse is an integrated cloud-native GCP platform to store, search, organize, govern and analyze documents and their structured metadata." \
  --release-level=preview \
  --transport=grpc \
  --language=java \
  --monorepo-url="https://github.com/googleapis/google-cloud-java"`

Reply:
Sorry, I found that the product doc is the bottom of this markdown file. (This is a style of how to write links)

https://screenshot.googleplex.com/98DoTbwLPnLitNh